### ISSUE TITLE
Question: What are proper parameters for crawler example to train ?

### ISSUE THREAD

**Issue by cerushdope** [Unity-Technologies/ml-agents#323]
Hello!
It's urgent because I have deadline, so please answer asap. I've been trying to train crawler example but it doesn't train well. I guess  That's all because of wrong parameters. Please suggest me good parameters for crawler (At least the ones that were used for learning the default model). @awjuliani @vincentpierre   and guys, it would be great if you provided best sets of parameters for each example. Thanks in advance.

**Comment by vincentpierre** [comment]
I am not sure what parameters were used for this one anymore. We left it so people will be able to explore and test new algorithms. From what I remember you need:
 * At least 10 crawlers
 * Large batch size (I think 1024)
 * Large buffer size (10240)
 * Low learning rate (1e-4)
 * Normalize true
 * At least 1e7 steps (I remember it took well over 24h to train)   

For 0.3, we will give training parameters for each demo environment.

**Comment by Lakrix** [comment]
@vincentpierre  When will 0.3 be released? Looking forward to it.
